src/components/layout/RegisterNameForm/RegisterNameForm.tsx

```diff
@@ -73,133 +67,260 @@ function RegisterNameForm() {
-              validityCallback={(isValid: boolean) => setIsValidPDNT(isValid)}
-              wrapperClassName={'flex flex-column center'}
-              wrapperCustomStyle={{ gap: '0.5em', boxSizing: 'border-box' }}
-              showValidationIcon={true}
-              inputClassName={'data-input center'}
-              inputCustomStyle={
-                isValidPDNT && pdntTxID
-                  ? { border: 'solid 2px var(--success-green)', width: '100%' }
-                  : !isValidPDNT && pdntTxID
-                  ? { border: 'solid 2px var(--error-red)', width: '100%' }
-                  : {}
-              }
-              placeholder={'Enter an PDNT Contract ID Validation Input'}
-              validationPredicates={{
-                [VALIDATION_INPUT_TYPES.ARWEAVE_ID]: {
-                  fn: (id: string) => arweaveDataProvider.validateArweaveId(id),
-                },
-                [VALIDATION_INPUT_TYPES.PDNT_CONTRACT_ID]: {
-                  fn: async (id: string) => {
-                    const txId = new ArweaveTransactionID(id);
-                    const state =
-                      await arweaveDataProvider.getContractState<PDNTContractJSON>(
-                        txId,
-                      );
-                    const contract = new PDNTContract(state, txId);
-                    return new Promise((resolve, reject) =>
-                      contract.isValid()
-                        ? resolve(id)
-                        : reject('PDNT contract has invalid schema'),
-                    );
-                  },
-                },
-                [VALIDATION_INPUT_TYPES.TRANSACTION_CONFIRMATIONS]: {
-                  fn: (id: string) =>
-                    arweaveDataProvider.validateConfirmations(id),
-                },
+        <div className="flex flex-column flex-center">
+          <div
+            className="flex flex-column flex-center"
+            style={{
+              width: '100%',
+              height: 'fit-content',
+              gap: '25px',
+            }}
+          >
+            <div
+              className="flex flex-row flex-space-between"
+              style={{ gap: '25px' }}
+            >
+              <button
+                className="flex flex-row center text-medium bold pointer"
+                onClick={() => setTransactionType(TRANSACTION_TYPES.LEASE)}
+                style={{
+                  position: 'relative',
+                  background:
+                    transactionType === TRANSACTION_TYPES.LEASE
+                      ? 'var(--text-white)'
+                      : '',
+                  color:
+                    transactionType === TRANSACTION_TYPES.LEASE
+                      ? 'var(--text-black)'
+                      : 'var(--text-white)',
+                  border: 'solid 1px var(--text-white)',
+                  borderRadius: 'var(--corner-radius)',
+                  height: '56px',
```

Comment:
this is showing as larger than the parent div and creating a small scroll affect (hover over buy and scroll)

Reply:
unable to reproduce, can you record the issue?